Source/Paths/SvgPathBuilder.cs

```diff
@@ -26,7 +26,7 @@ public class SvgPathBuilder : TypeConverter
         /// <param name="path">A <see cref="string"/> containing path data.</param>
         public static SvgPathSegmentList Parse(string path)
         {
-            if (string.IsNullOrEmpty(path))
+            if (path == null)

```

Comment:
Did you encounter this condition?

Reply:
@mrbean-bremen 

Thank you for your review.

The following SVG file:
```svg
<?xml version="1.0" encoding="UTF-8"?>
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="10" height="10">
  <path d="" opacity="0.5" />
</svg>
```